askama_shared/src/generator.rs

```diff
@@ -210,15 +210,13 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
     // Implement Actix-web's `Responder`.
     fn impl_actix_web_responder(&mut self, buf: &mut Buffer) -> Result<(), CompileError> {
         self.write_header(buf, "::actix_web::Responder", None)?;
-        buf.writeln("type Future = ::askama_actix::futures::Ready<::std::result::Result<::actix_web::HttpResponse, Self::Error>>;")?;
-        buf.writeln("type Error = ::actix_web::Error;")?;
         buf.writeln(
             "fn respond_to(self, _req: &::actix_web::HttpRequest) \
-             -> Self::Future {",
+             -> actix_web::HttpResponse {",
```

Comment:
Ah, sorry I missed this before, but this really needs the `::` prefix before `actix_web` so that we don't rely on `actix_web` being in scope in every module containing a template.

Reply:
ok